client/devdocs/style.scss

```diff
@@ -140,6 +140,31 @@
 	margin-top: 48px;
 }
 
+.design-assets {
+	@include breakpoint( "<660px" ) {
+		padding: 0 6px;
+	}
+}
+
+.design-assets h2,
```

Comment:
Nope, it is. I hacked around it in https://github.com/Automattic/wp-calypso/pull/6458/files